opencti-platform/opencti-graphql/src/domain/stixCyberObservable.js

```diff
@@ -147,7 +148,11 @@ export const createIndicatorFromObservable = async (context, user, input, observ
     externalReferences: input.externalReferences,
     update: true,
   };
-  return await addIndicator(context, user, indicatorToCreate);
+};
+
+export const createIndicatorFromObservable = async (context, user, input, observable) => {
```

Comment:
Code look better than before but unrelated to background tasks, is it a big fix ?